src/token.ts

```diff
@@ -185,6 +185,15 @@ export function isExpired(ucan: Ucan): boolean {
   return ucan.payload.exp <= Math.floor(Date.now() / 1000)
 }
 
+/**
+ * Check if a UCAN is not valid yet.
+ *
+ * @param ucan The UCAN to validate
+ */
+export const isNotValidYet = (ucan: Ucan): boolean => {
+  return ucan.payload.nbf > Math.floor(Date.now() / 1000)
+}
```

Comment:
@bgins could we phrase this in the positive? Otherwise we can end up with `!isNotValidYet(foo)` and `isNotValidYet(foo) ? x : y`... it gets confusing

Reply:
Yeah, agreed. The double negative is less than ideal.

I'm not quite sure how to name this in the positive. Here are some possibilities that we could consider:

- `hasBecomeValid`. Wordy, and a little awkward.
- `isLatent`. Latent may not be a familiar enough term.
- `isDormant`. Seems correct, but less technical.
- Something else?

Reply:
I like `hasBecomeValid` :)

Reply:
I've updated the helper function to `isTooEarly`. This name avoids the double negative and better indicates time than the other names we came up with.

I also updated the comment on the function to "Check if a UCAN is not active yet". I think this will make more sense in context when we add an `isActive` helper function, but I'd be open to wording this differently.